src/sentry/sentry_metrics/visibility/metrics_blocking.py

```diff
@@ -7,11 +7,17 @@
 from sentry.sentry_metrics.visibility.errors import MalformedBlockedMetricsPayloadError
 from sentry.utils import json
 
-BLOCKED_METRICS_PROJECT_OPTION_KEY = "sentry:blocked_metrics"
+METRICS_BLOCKING_STATE_PROJECT_OPTION_KEY = "sentry:blocked_metrics"
```

Comment:
Kept the option the same to avoid issues. Even though, no usage of the endpoint should have happened by now.